src/components/input/text_input.module.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from "./text_input.module.scss";
+
+
+const TextInput: React.FC<Props> = ({
+  placeholder,
+  value,
+  onChange,
+  size = "normal",
+}) => {
+
+  return (
+    <input
+      type={"search"}
```

Comment:
`type` should be a prop which defaults to `text`, otherwise we can't use `email` or `password` inputs without duplicating this component